### PR TITLE
Test on Django 4.2

### PR DESCRIPTION
**Pull request by adamchainz**

### CODE REVIEW

File: flags/urls.py [hunk 1]
```diff
@@ -78,7 +78,6 @@ def url_patterns(self):
                 pattern.default_args,
                 pattern.name,
             )
-            print(pattern, flagged_pattern)
```
Comment (adamchainz) [inline flags/urls.py:81]: Woops! This was printing in tests and for any users. Added in #101.

Comment (willbarton) [reply flags/urls.py:81]: Good catch!

**Review by willbarton — APPROVED** [review]

**Comment by willbarton** [comment]
Rebased and merged as https://github.com/cfpb/django-flags/commit/c95721e559bcc4fbfaa30e007e4a9ec2d7a17218

**Comment by willbarton** [comment]
Rebased and merged as https://github.com/cfpb/django-flags/commit/2667995ce649914a0bfecc0b18e1f42f9515b70a

**Pull request closed without merging**